gensrc/thrift/FrontendService.thrift

```diff
@@ -623,6 +623,14 @@ struct TStreamLoadPutRequest {
     31: optional string merge_condition
     // only valid when file type is CSV
     50: optional string rowDelimiter
+    // only valid when file type is CSV
```

Comment:
1. can they be encapuslated in a struct, and reused somewhere?
2. why not keep the same naming format like xxx_yyy?